src/simplewallet/simplewallet.cpp

```diff
@@ -442,6 +442,21 @@ namespace
       fail_msg_writer() << tr("unexpected error: ") << e.what();
     }
   }
+
+  bool check_file_overwrite(const std::string &filename)
+  {
+    if (boost::ends_with(filename, ".keys"))
+    {
+      fail_msg_writer() << boost::format(tr("File %s stores wallet private keys! Use a different file name.")) % filename;
+      return false;
+    }
```

Comment:
Why single this out ?

Reply:
Because I thought those `.keys` files which likely contain Monero wallet keys may be of much higher importance and deserve a special warning message. Could be dropped if deemed unnecessary, though.